src/tokentype.js

```diff
@@ -109,6 +109,7 @@ export const types = {
   starstar: new TokenType("**", {beforeExpr: true}),
 
   // Keyword token types.
+  _await: kw("await"),
```

Comment:
The spec for async/await doesn't say anything about it being a keyword though, so I think it's better to treat it the same way we treat other contextual keywords.